### ISSUE TITLE
[AC-16] Detect browser timeZone if none specified and support short TZ like HST for Pacific/Honolulu

### ISSUE THREAD

**Issue by clouless** [codeclou/advanced-codeblocks#16]
# Bug

Confluence specifies timezones either per User or per Server (if user has not selected one).
There are special timezones that come from Java that JavaScript does not understand such as HST for Hawaiian Standard Time.
Also should the timezone be detected automatically (browser) if none is specified.

# Acceptance Criteria

 * ✅ All special timezones provided by Confluence are understood by JavaScript and converted to standard IANA timezones internally
```
HST  (GMT-1000) Hawaii                              => Pacific/Honolulu
AST  (GMT-0900) Alaska                              => America/Anchorage
GMT  (GMT+0000) Universal Coordinated Time (UTC)    => Etc/UTC
ECT  (GMT+0100) West-Zentralafrika (WAT)            => Africa/Lagos
```
 * ✅ the timezone is detected automatically (browser) if none is specified

# Test Scenario

**GIVEN: ACR Macro exists in page and timezone is Europe/Berlin**
 * A page with the ACR (Advanced Remote Codeblock) Macro and refreshed cache exists
 * User timezone is Europe/Berlin
 * <img width="1521" alt="Bildschirmfoto 2019-11-16 um 12 30 01" src="https://user-images.githubusercontent.com/12599965/68992545-1ed5b180-086d-11ea-873d-fd8dff788005.png">

**WHEN: User switches timezone to Hawaii**
 * Switch the user Timezone to Hawaii and refresh the page containing the ACR Macro

**THEN: Date on page should be displayed in Pacific/Honolulu timezone**
 * <img width="1521" alt="Bildschirmfoto 2019-11-16 um 12 27 04" src="https://user-images.githubusercontent.com/12599965/68992569-5e040280-086d-11ea-8c98-385dd11ba3f4.png">

**Comment by clouless** [comment]
### Internal Issues

 * [ACF-INT-10] Detect browser timeZone if none specified and support short TZ like HST for Pacific/Honolulu

**Comment by clouless** [comment]
Released to Marketplace in v3.2.0 on 2019-11-16
